src/alexjs/checkBannedWords.ts

```diff
@@ -12,11 +12,13 @@ export const checkBannedWords = async (
 ): Promise<EmbedBuilder[]> => {
   const embeds: EmbedBuilder[] = [];
   try {
+    const urlPattern = /https?:\/\/[^\s]+/g;
```

Comment:
Could we use the existing link regex check from the other file? Maybe move to the config?

https://github.com/EddieHubCommunity/EddieBot/blob/9fc6b257c48b594fb3a48b421cccfabbef6a7a08/src/links/checkLinks.ts#L7-L8

Reply:
Sure

Reply:
What about moving it to a reusable config, so we don't have duplication? `urlRegex.js`

https://github.com/EddieHubCommunity/EddieBot/tree/9fc6b257c48b594fb3a48b421cccfabbef6a7a08/src/config